src/http_server.cc

```diff
@@ -235,8 +236,24 @@ HTTPServer::Start()
 }
 
 TRITONSERVER_Error*
-HTTPServer::Stop()
+HTTPServer::Stop(uint32_t* exit_timeout_secs, const std::string& service_name)
 {
+  {
+    std::lock_guard<std::mutex> lock(conn_mu_);
+    accepting_new_conn_ = false;
+  }
+  if (exit_timeout_secs != nullptr) {
+    while (*exit_timeout_secs > 0) {
```

Comment:
```suggestion
    while ( (*exit_timeout_secs > 0) && (conn_cnt_>0)) {
```

Reply:
small tweak - I think you can make this a little simpler by just checking for both conditions in the while loop:
```
while ( (*exit_timeout_secs>0) &&  (conn_cnt_>0)) {
  std::this_thread::sleep_for(std::chrono::seconds(1)));
 (*exit_timeout_secs)--;
}
```

The logic is a little easier to follow then a break in the loop - it becomes a natural condition for exit

Reply:
sure, updated: [Group checking logic](https://github.com/triton-inference-server/server/pull/6986/commits/6dbd249c65215add12c900362c7a16b2c683fa34)